src/web/step_definitions/elements.ts

```diff
@@ -160,3 +159,15 @@ When(/^I press the "([^"]*)" key$/, key => {
 When(/^I drag "([^"]*)" element and drop over "([^"]*)" element$/, function(elementDrag, elementDrop) {
   return methods.interactions.dragAndDrop(this.currentPage, elementDrag, elementDrop);
 });
+
+When(/^I confirm the recaptcha$/, function() {
+  const iframeElement = '.g-recaptcha iframe';
+  const iframeClickArea = '#rc-anchor-container';
+  const iframeConfirmation = '#recaptcha-anchor.recaptcha-checkbox-checked';
+
+  return this.currentPage
```

Comment:
Two things:
1. I would like to have this method inside `/src/web/methods` and just imported here. 
2. How many times did you check if the recaptcha is properly "checked"? I had a problem in the previous project and I had to add wait for 500ms after the `waitForVisibilityOf` action is finished

Reply:
1. Done
2. About ~30 times. Maybe instead of hardcoded wait value we should add a custom wait in case if someone has that problem?

Reply:
I think it's still not done. It should look pretty much the same as the step above.